mitodl/mitxonline
# Add prerequisites to courses API

### What are the relevant tickets?
https://github.com/mitodl/hq/issues/4947

### Description (What does it do?)
Adds a required_prerequisites field to the courses api

### How can this be tested?
Make a GET call to the courses API for a course with nothing defined in the prerequisite field of the course page in the CMS.  Verify that the required_prerequisites in the API response is false.

Make a GET call to the courses API for a course with text defined in the prerequisite field of the course page in the CMS.  Verify that the required_prerequisites in the API response is true.
